chocolatey/choco
# (#2641) Add deprecation messages for shims

## Description Of Changes

This pull request updates the help text and documentation around the commands to mention that the shims we have available is now deprecated and will be removed in v2.0.0.
Additionally, a missed location mentioning that the search command is an alias for list is removed as these will be considered two seperate commands in v2.0.0.

## Motivation and Context

To do our best to give users the information needed for future usage of the Chocolatey executable.

## Testing

1. Run `choco -?` and notice there is a deprecation warning about shims going away.
2. Run `choco install -?` and a mention of the shim `cinst` is deprecated will be shown.
3. Run `choco list -?` and the same mention is shown
4. Run `choco search -?` and notice there are no deprecation notice about the shim, however there is a notice in the example of it.
5. Run `choco push -?` and the same deprecation notice should be shown.
6. Run `choco uninstall -?` and the same here.
7. Run `choco upgrade -?` and this is also the same.
8. Generate the documentation using `GenerateDocs`, and verify all commands have a warning that the shims are deprecated and will be removed.


## Change Types Made



## Related Issue


fixes #2641  
Small change for #2462  
https://app.clickup.com/t/20540031/ENGTASKS-723

## Change Checklist
